draft-ietf-ohai-ohttp.md

```diff
@@ -1124,6 +1130,14 @@ relay for managing denial of service; see {{dos}}. It is also necessary to
 provide confidentiality protection for the unprotected requests and responses,
 plus protections for traffic analysis; see {{ta}}.
 
+Unsecured requests - such those with the "http" scheme as opposed to the "https"
+scheme - MUST NOT be used if the Oblivious Gateway and Target Resources are
+operated by different entities as that would expose both requests and response
+to modification or inspection by a network attacker.
```

Comment:
```suggestion
Nonsecure requests - such those with the "http" scheme as opposed to the "https"
scheme - SHOULD NOT be used if the Oblivious Gateway and Target Resources are
operated by different entities as that would expose both requests and response
to modification or inspection by a network attacker.
```

Reply:
I just realized that maybe OHTTP could be used for CRL fetching, which a MUST NOT would exclude.  I'd still prefer something stronger, but I can live with this.

Reply:
Bingo!

Reply:
Citing CRL and OCSP queries as examples where the SHOULD is not followed seems like a good thing to do.